Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "categoricalbar"
+      },
+      "name": "query - 3"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Disk Utilization"
+      },
+      "name": "text - 4"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SapHana_LoadHistory_CL\r\n| summarize avg(DISK_USED_d) by HOST_s , bin(UTC_TIMESTAMP_t, 15m)",
+        "size": 0,
+        "color": "turquoise",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "areachart"
+      },
+      "name": "query - 5"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Actual CPU vs Forecast 7 Days"
+      },
+      "name": "text - 8"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_LoadHistory_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgCPU=avg(CPU_d) default=0 on UTC_TIMESTAMP_t in range(ago(7d),projectTo, 1h) by HOST_s\r\n| extend SizeForecast = series_decompose_forecast(avgCPU, projectForward*24)",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
+        ],
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "Memory % Used  vs Forecast 7 Days"
+      },
+      "name": "text - 12"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let startDate = startofday(ago(7d)); // go back in time nn days\r\nlet endDate = now(); // what is the date now\r\nlet projectTo = now()+7d; // project forward nn days\r\nlet projectForward = 7; // must be same as projectTo value\r\nSapHana_Infra_CL\r\n|where TimeGenerated > ago(7d)\r\n| make-series avgMEM=avg(MEMORY_USED_d/MEMORY_SIZE_d)*100 default=0 on TimeGenerated in range(ago(7d),projectTo, 1h) by HOST_s\r\n| extend SizeForecast = series_decompose_forecast(avgMEM, projectForward*24)",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "linechart",
+        "graphSettings": {
+          "type": 0
+        }
+      },
+      "name": "query - 7 - Copy"
+    },
+    {
+      "type": 1,
```

Comment:
you're using a lot of text steps as titles.  the query steps can have a title instead, see chart title in advanced settings in the query step.

Reply:
Will build that into the "real" template.